Added some extra info to README about doing a hotfix/patch AB#1742 (#693)
global-121/121-platform: README.md

  - [ ] Download the generated artifacts (`AW-App.zip` and `PA-App.zip`)
  - [ ] Rename to match the version (i.e: `PA-App-v0.1.0.zip`)
- [ ] "[Draft a release](https://github.com/global-121/121-platform/releases/new)" on GitHub  
      - Select the new `release`-branch.
      - Add a short description and/or link to relevant other documents.
      - Create/publish the release on GitHub.
      - Add a new tag as <version>.

### Patches/Hotfixes Checklist

> Doing patchers/hotfixes follows the same process as the release branches. With some small changes.

Reviewer: ```suggestion
Doing patchers/hotfixes follows the same process as a regular release With some small changes.
```

> - Code does not need to be freezed.

Reviewer: ```suggestion
- Code does not need to be frozen. (As there is no active development on the release-branch)
```

- Checkout from the `release/<version>`-branch that needs a hotfix.

> - The update to the Changelog should only be committed to the new hotfix release branch.

Reviewer: ```suggestion
- The update to the Changelog should only be committed to the new hotfix release branch.
- After the hotfix-release, apply the same fix (if applicable) to the master-branch in a regular PR
- Add the hotfix-release to the CHANGELOG in the master-branch
```
